src/js/components/SchemaForm.js

```diff
@@ -181,9 +193,13 @@ class SchemaForm extends React.Component {
     );
   }
 
-  getServiceHeader() {
+  getFormHeader() {
```

Comment:
This is the header of the advanced package install modal. And that header ain't needed in the deploy service modal.